frontend/packages/shared/src/components/altinnHeader/AltinnHeader.tsx

```diff
@@ -0,0 +1,56 @@
+import { useAppSelector } from 'app-development/hooks';
+import { useUserQuery } from 'app-development/hooks/queries/useUserQuery';
+import AltinnStudioLogo from 'app-shared/navigation/main-header/AltinnStudioLogo';
+import { ProfileMenu } from 'app-shared/navigation/main-header/profileMenu';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import classes from './AltinnHeader.module.css';
+import { AltinnSubMenu } from '../altinnSubHeader';
+import { AltinnHeaderMenu } from '../altinnHeaderMenu';
+import { AltinnHeaderButtons } from '../altinnHeaderButtons';
+
+export interface IAltinnHeader {
```

Comment:
Konvensjonen er å kalle interfacer som definerer input til en komponent for `[komponentnavn]Props`. `I`-prefikset er heller ikke nødvendig, selv om  det brukes ukonsekvent flere steder i Studio (vi burde med andre ord fjerne det overalt). Så da blir det `AltinnHeaderProps` i dette tilfellet.

Reply:
Endret